### PR TITLE
[KOA-4489] Fix snapshot testing status bar

### PR DESCRIPTION
**Pull request by georgegillams**
Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/main/Example/Backpack%20Screenshot/Screenshots.swift)
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The _screenshots_ folder stores all of the screenshots we use on the [documentat
 bundle exec rake take_screenshots
 ```
 
-The script takes 10-15 minutes. It also requires that the iPhone Simulator is not already running.
```
Comment (georgegillams) [inline CONTRIBUTING.md:58]: Fastlane will boot the simulator if it's not running, so this is incorrect.

File: Example/Backpack Screenshot/SnapshotHelper.swift [hunk 2]
```diff
@@ -165,7 +165,11 @@ open class Snapshot: NSObject {
             }
 
             let screenshot = XCUIScreen.main.screenshot()
+            #if os(iOS)
```
Comment (georgegillams) [inline Example/Backpack Screenshot/SnapshotHelper.swift:168]: All the changes in here are made by fastlane

File: Rakefile [hunk 3]
```diff
@@ -114,9 +114,7 @@ task :take_screenshots do
   # Remove existing screenshots
   FileUtils.rm_rf('screenshots')
 
-  sh "xcrun simctl boot \"#{TEST_DEVICE_NAME}\""
```
Comment (georgegillams) [inline Rakefile:117]: Fastlane will boot the simulator if it's not running, so this is not needed.

File: Rakefile [hunk 4]
```diff
@@ -114,9 +114,7 @@ task :take_screenshots do
...
-  sh "xcrun simctl status_bar \"#{TEST_DEVICE_NAME}\" override --time \"09:41\" --wifiBars 3 --cellularBars 4 --batteryLevel 100 --operatorName Backpack"
```
Comment (georgegillams) [inline Rakefile:118]: We now do this on app launch instead of here

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**